js/src/pages/price-benchmark/banner/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { Button, Card, CardBody } from '@wordpress/components';
+import { createInterpolateElement, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Pill } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { usePreferences } from '~/hooks/usePreferences';
+import bannerImage from '~/images/price-benchmark/price-benchmark-banner-icon.svg';
```

Comment:
```suggestion
import bannerImageURL from '~/images/price-benchmark/price-benchmark-banner-icon.svg';
```

Just to be consistent with the codebase.